Settings.h

```diff
@@ -26,6 +28,7 @@ typedef struct {
 
 typedef struct Settings_ {
    char* filename;
+   int config_version;
```

Comment:
This variable is never read from.

Reply:
It is set and made available when we have the next incompatible change to fence that one.